### ISSUE TITLE
About validation

### ISSUE THREAD

**Issue by onetance** [CCIIPLab/GCE-GNN#25]
Excuse me. Does the model that get the results recorded in the paper use the validation set for training? Are the hyperparameters obtained on the validation set first, and then the model is retrained on the training set and validation set? I can't seem to get the results in the paper.

**Comment by fgriver** [comment]
I also meet this question, cannot reproduce the results in paper

**Comment by onetance** [comment]
> I also meet this question, cannot reproduce the results in paper

So how do you solve this problem

**Comment by CCIIPLab** [comment]
Thank you for your attention to our work. 
You can directly use the hyperparameters provided in our code (which we obtained through searching) to run our code. Additionally, we believe that if there are some minor differences from the results in our paper, it may be due to randomness.
